services/product/src/handlers/catalogBatchProcess.ts

```diff
@@ -0,0 +1,66 @@
+import 'source-map-support/register';
+import {SQSEvent} from 'aws-lambda';
+import {SNS} from 'aws-sdk';
+import Logger from 'bunyan';
+import {Connection} from "typeorm";
+
+import {createResponse} from '../../../../utils';
+import {RESPONSE_CODES} from '../../../../constants';
+import {Product} from '../entity';
+import {REGION} from '../config';
+import {createProductTransaction} from './createProduct';
+
+const {
+    SNS_TOPIC_SUCCESS_ARN,
+    SNS_TOPIC_ERROR_ARN,
+} = process.env;
+
+export const catalogBatchProcess = async (event: SQSEvent, connection: Connection, logger: Logger) => {
+    const sns = new SNS({
+        region: REGION,
+    });
+    const promises = event.Records.map( async record => {
+        logger.info('new record:', record.body);
+        try{
+            const {
+                count, ...product
+            } = JSON.parse(record.body);
+            const result = await createProductTransaction(product as Product,count, connection);

```

Comment:
I think the main idea was to send a post request to 'createProduct' lambda and then that lambda should write the product to DB.

Reply:
I'm not sure.

> Create a SQS queue, called catalogItemsQueue, in the resources section in serverless.yml file. Configure the SQS to trigger lambda catalogBatchProcess with 5 messages at once via batchSize property. The lambda function should iterate over all SQS messages and create corresponding products in the products table.
> 
> Update the importFileParser lambda function (TASK 5) to send each CSV record into SQS.